src/test/javascript/spec/component/plagiarism/plagiarism-sidebar.component.spec.ts

```diff
@@ -84,13 +85,30 @@ describe('Plagiarism Sidebar Component', () => {
     });
 
     it('should handle changes correctly', () => {
```

Comment:
This tests the pagination. Please rename the test accordingly

Reply:
@julian-christl Good call, I renamed the test case now 👍🏼